client/extensions/woocommerce/app/settings/taxes/taxes-rates.js

```diff
@@ -0,0 +1,323 @@
+						{
+							components: { strong: <strong /> }
+						}
+						) }
+				</Notice>
+			);
+		}
+
+		if ( DESTINATION_BASED_SALES_TAX === stateData.salesTaxBasis ) {
+			return (
+				<p>
+					{ translate( 'We\'ll automatically calculate and charge the ' +
+						'correct rate of tax for you each time a customer checks out.' ) }
+				</p>
+			);
+		}
+
+		return (
+			<p>
+				{ translate( 'We\'ll automatically calculate and charge sales tax ' +
+					'at the following rate each time a customer checks out.' ) }
+			</p>
+		);
+	}
+
+	possiblyRenderRates = () => {
+		const { address, areTaxesEnabled, taxRates, translate } = this.props;
+		if ( ! areTaxesEnabled ) {
+			return null;
+		}
+
+		const stateData = getStateData( address.country, address.state );
+		if ( ! stateData ) {
+			return null;
+		}
+
+		if ( ORIGIN_BASED_SALES_TAX !== stateData.salesTaxBasis ) {
+			return null;
+		}
+
+		if ( isEmpty( taxRates ) ) {
+			return (
+				<Notice showDismiss={ false } status="is-error">
+					{ translate( 'The WordPress.com sales tax rate service is not ' +
+						'available for this site.' ) }
+				</Notice>
+			);
+		}
+
+		let state_rate = ( 'state_rate' in taxRates ) ? parseFloat( taxRates.state_rate ) : 0;
+		let combined_rate = ( 'combined_rate' in taxRates ) ? parseFloat( taxRates.combined_rate ) : 0;
+		let local_rate = combined_rate - state_rate;
+
+		state_rate = round( state_rate * 100, 2 );
+		combined_rate = round( combined_rate * 100, 2 );
+		local_rate = round( local_rate * 100, 2 );
+
+		const rates = [];
+
+		if ( 0 < state_rate ) {
+			rates.push( {
+				name: translate( 'State Tax' ),
+				rate: state_rate
+			} );
+			if ( 0 < local_rate ) {
+				rates.push( {
+					name: translate( 'Local Tax' ),
+					rate: local_rate
+				} );
+			}
+		}
+
+		rates.push( {
+			name: translate( 'Total Tax' ),
+			rate: combined_rate
+		} );
+
+		return (
+			<Table>
+				<TableRow isHeader>
+					<TableItem isHeader>
+						{ translate( 'Name' ) }
+					</TableItem>
+					<TableItem isHeader>
+						{ translate( 'Rate' ) }
+					</TableItem>
+				</TableRow>
+				{ rates.map( ( row, i ) => (
+					<TableRow key={ i }>
+							<TableItem key={ i }>{ row.name }</TableItem>
+							<TableItem key={ i + 1 }>{ row.rate }%</TableItem>
+					</TableRow>
+				) ) }
+			</Table>
+		);
+	}
+
+	render = () => {
+		const { loadedSettingsGeneral, loadedTaxRates, onEnabledChange, taxesEnabled, translate } = this.props;
+
+		if ( ! loadedSettingsGeneral ) {
+			return null;
+		}
+
+		if ( ! loadedTaxRates ) {
+			return null;
+		}
+
+		const toggleMessage = taxesEnabled ? translate( 'Tax calculations enabled' )
+			: translate( 'Tax calculations disabled' );
+
+		return (
+			<div className="taxes__taxes-rates">
+				<ExtendedHeader
+					label={ translate( 'Tax rates' ) } >
+					<FormToggle
+						name="taxesEnabled"
+						onChange={ onEnabledChange }
+						checked={ taxesEnabled } >
+						<span>
+								{ toggleMessage }
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/15541/commits/17f50964f1f6156641032d53a6b96785ad2895a0